suite/tests/linux/rseq.c

```diff
@@ -1141,10 +1151,9 @@ kernel_xfer_event(void *drcontext, const dr_kernel_xfer_info_t *info)
 #        error Unsupported arch
 #    endif
 #    ifdef DEBUG /* See above: special code in core/ is DEBUG-only> */
-        /* Check that the interrupted PC for the true abort case is *prior* to the
-         * committing store.
-         */
-        assert(info->source_mcontext->pc == (app_pc)test_rseq_native_abort_pre_commit);
+        /* Check that the interrupted PC for the true abort case is the handler. */
+        assert(!in_rseq_migration_test ||
```

Comment:
Since we're inside the if-body, in_rseq_migration_test is true always.

Reply:
Removed.